package.json

```diff
@@ -62,6 +65,9 @@
     "sinon": "~4.1.3"
   },
   "dependencies": {
+    "dgrid": "^1.2.1",
```

Comment:
Do we want to pin these? Does this mean we only support this version range of dgrid?

Reply:
I'll pin them. Hmm, we're using the dgrid APIs so I'd think we support any version that doesn't have breaking changes in those APIs.